app.js

```diff
@@ -186,3 +190,21 @@ function syncToTarget (event, target) {
     server.replicateNetwork(socket, 'pull')
   }
 }
+
+function getGlobalDatasetCentroid (done) {
+  var bbox = [[-90,90],[-180,180]]
+
+  var stream = osm.queryStream(bbox)
+  var ix = setInterval(function () {
```

Comment:
Is there another way? Doing this in a synchronous loop will block the node process.